docker/Dockerfile

```diff
@@ -84,10 +77,38 @@ RUN curl -L -k -o "${PWD}/pnetcdf.tar.gz" \
             make install && \
             rm -rf "${PWD}/pnetcdf"
 
-RUN mkdir -p /root/.cime /storage/timings
+# CESM dependencies
```

Comment:
Is this the only place we need to update dependencies for cesm/cime?

Reply:
Should they be passed in as the external dependancies (eg ESMF_VERSION) are?

Reply:
Yes this is the only place you need to update dependencies for cesm/cime.

I can update these to build arguments as well but I would like to **always** have the defaults specified in the Dockerfile and not the workflow.

Reply:
If I understand you correctly then you should not make them arguments.